packages/nested-clients/src/index.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @internal
+ */
+export { CreateTokenCommand, SSOOIDCClient } from "./nested-sso-oidc/index";
```

Comment:
Should we export sso under `@aws-sdk/nested-clients/sso`?
That way a conflict between commands can be avoided.

Reply:
sounds good, I can port the submodule system from core over here. I'll hold off on merging for now.